src/utils/system.py

```diff
@@ -89,15 +89,15 @@ def binary_to_human(n: int) -> str:
     """
     Convert bytes or bits into human readable format with binary prefix
     """
-    symbols = ("K", "M", "G", "T", "P", "E", "Z", "Y")
+    symbols = ("M", "G", "T", "P", "E", "Z", "Y")
     prefix = {}
     for i, s in enumerate(symbols):
         prefix[s] = 1 << (i + 1) * 10
     for s in reversed(symbols):
         if n >= prefix[s]:
             value = float(n) / prefix[s]
-            return "%.1f%s" % (value, s)
-    return "%s" % n
+            return "%d%s" % (value, s)
```

Comment:
hmmm, shouldnt we keep `@.1f` here ? Not sure what it means exactly, I'm suspecting this is to have stuff displayed as e.g. `12.3M` ?

Reply:
I replaced with %d to display the rounded number *12M*, since K has no decimal point, a kind of harmonization, like in roundcube value are rounded. 

But we can keep the %.1f, if you don't want result to be rounded to unit.